pjsip/src/pjsip-ua/sip_inv.c

```diff
@@ -502,6 +502,9 @@ static pj_status_t inv_send_ack(pjsip_inv_session *inv, pjsip_event *e)
      */
     if (inv->state < PJSIP_INV_STATE_CONFIRMED) {
 	inv_set_state(inv, PJSIP_INV_STATE_CONFIRMED, &ack_e);
+    } else if (inv->state == PJSIP_INV_STATE_DISCONNECTED) {
```

Comment:
please put some info/comment or simply a link to this PR for future ref.

btw, the issue is reproducible and the patch does fix it, right?
also, I've found another scenario that may lead to leaked tdata, will add more info & perhaps patch later.

Reply:
Yes, it's reproducible and the patch fix it.